app/Http/Controllers/EventCheckoutController.php

```diff
@@ -362,7 +362,7 @@ public function showEventPayment(Request $request, $event_id)
             'tickets' => $order_session['tickets'],
             'order_total' => $order_total,
             'orderService' => $orderService,
-            'order_requires_payment'  => (ceil($order_session['order_total']) == 0) ? false : true,
+            'order_requires_payment'  => ceil($order_session['order_total']) > 0,
```

Comment:
I'd prefer to keep this as ternary operator. Reason is in the same file for the same condition a ternary operator is being used. 
https://github.com/Attendize/Attendize/blob/master/app/Http/Controllers/EventCheckoutController.php#L211

Prefer to not add more inconsistencies to an already inconsistent codebase.

Reply:
Okay, so while we're at it... Why not do it right?

Check my last commit to see if you think it's right:

- Remove (a lot) of duplicate code
- It avoids the cyclomatic complexity of the ternary operator, which is certainly difficult to read.
- Add some helpers to use it in Blade
- Apart from that, it improves the check to support negative numbers, i.e. if the order is -11$ it will say that there is no need to pay it.

We have to do this many times with a thousand more things, but I think it's a good starting point.